### DISCUSSION TITLE
Celebrating 10 years of FreshRSS 🎂

### DISCUSSION THREAD

**Discussion by marienfressinaud** [FreshRSS/FreshRSS#4667] [General]
(English)

Dear community,

This month, FreshRSS will celebrate its 10th anniversary (cf. [the first commit](https://github.com/FreshRSS/FreshRSS/commit/fb57be5a5af3a2fb46b2dbf2b503ffe78eb5cf49)). For the occasion, we have redesigned [our website](https://freshrss.org), we hope you like it!

Now, we would like to hear you: how do you like FreshRSS? For how long have you been using it? How did it help you over the years? What are you hoping for for the future? **This discussion is opened so you can express yourself!**

We want to thank you for using FreshRSS, it would never have made it this long without the community!

---

(Français)

Chère communauté,

Ce mois-ci, FreshRSS va fêter ses 10 ans (cf. [le premier commit](https://github.com/FreshRSS/FreshRSS/commit/fb57be5a5af3a2fb46b2dbf2b503ffe78eb5cf49)). Pour l’occasion, nous avons refait [notre site Web](https://freshrss.org), on espère que vous l’appréciez !

Désormais, nous aimerions vous entendre : que pensez-vous de FreshRSS ? Depuis combien de temps l’utilisez-vous ? Comment vous a-t-il aidé au fil des ans ? Qu’espérez-vous pour la suite ? **Cette discussion est ouverte pour que vous puissiez vous exprimer !**

Nous voulons vous remercier pour utiliser FreshRSS, il n'aurait jamais pu vivre aussi longtemps sans la communauté !

---

(other languages? feel free to translate the message above and I'll update this one :))

**Comment by rakoo** [comment]
Merci pour FreshRSS ! Je l'ai utilisé y'a un bail (y'a un peu moins de 10 ans en fait) et il fonctionnait super bien. J'ai arrêté de l'utiliser, non pas parce qu'il est mauvais, mais parce que je me suis rendu compte que je devenais accroc aux flux et me donnait une FOMO toxique. Mais si je devais réutiliser un agrégateur FreshRSS ferait clairement partie des choix que j'envisagerais !

**Reply by Alkarex** [reply]
Dans FreshRSS 1.20.0, il y a une nouvelle option pour marquer automatiquement comme lus les articles qui ne sont plus dans le flux du site d'origine. Combiné au fait qu'il y a aussi une option pour garder au maximum x articles non-lus par flux, cela permet de limiter le problème mentionné :-)

**Reply by rakoo** [reply]
C'est un début de solution mais ça ne remplace pas le problème initial : vérifier toutes les 5 minutes si y'a pas un truc nouveau. Mais c'est bon de savoir qu'il y a ce genre d'options :)

**Comment by fishyo** [comment]
I have been using freshrss built by myself for about half a year. Compared with other similar tools, I think it is the most suitable for me. I hope it can develop better and better, and thank the developers for their efforts!

**Comment by lapor-kris** [comment]
First, thanks for the development of FreshRSS. I've been using it now for about a year and it work great. I have zero issues and I'm loving it 😍

**Comment by ghost** [comment]
Bonjour

Et bien, j'utilise FreshRSS depuis qu'il est tout petit et il m'accompagne depuis, tous les jours. Chez moi comme au bureau, j'ai toujours un onglet ouvert dessus.

Il m'aide à faire ma veille technologique, suivre les blogs des copains, l'actu de mon quartier et les mises à jour de mes logiciels préférés sur GitHub. Bref, c'est mon outil d'information quotidienne depuis près d'une décennie.
J'avoue ne pas espérer grand chose parce qu'il me convient comme il est. 

Merci à vous, merci à la communauté. J'espère pouvoir l'utiliser encore longtemps et un jour, peut être, pouvoir y contribuer aussi. Merci !

**Comment by jidea** [comment]
Je viens de découvrir FreshRSS. Pour le moment il correspond à l'outil que je cherchais.

**Comment by Marc-Harony** [comment]
Bonjour 👋🏻 
J'ai découvert votre outil il y a peu de temps. Pour un projet d'entreprise je devqis choisir différentes solutions d'agrégation RSS.
J'ai choisi FreshRSS car la documentation est très claire, très détaillée et la commu très présente. Merci pour toute l'aide que vous m'avez apportée !
Bon courage pour la suite, je continuerais à me tenir informé sur les mises à jour !

**Comment by PrestonMcAfee** [comment]
Thanks for FreshRSS!  I used tinyRSS until a couple of years ago when a software update broke it and my site.  FreshRSS is much better and I have happily used it several times daily ever since.  Thanks to FreshRSS, I no longer miss greader.  The feedme app on Android also works well with FreshRSS.  Thank you and Merci!

**Comment by vladmycode** [comment]
I love this reader, specially the XPATH scraper feature - it really makes it possible to follow 99% of the website I need. Thanks a lot for FreshRSS! All the best!

**Comment by jeherve** [comment]
I only discovered FreshRSS a few months ago, and I'm really enjoying it so far. Checking my feeds has become part of my daily routine, just like back when I was still using Fever, more than 10 years ago. I find myself checking my RSS feeds instead of checking my Social Media profiles, and I think that's much better for my mental health. 

I'm also happy to be using RSS from a server that I control and own, thanks to FreshRSS.

Thank you team for all your hard work over the years! 🙇

**Comment by Fake4d** [comment]
I use FreshRSS since about 7 years of my life and it is a nearly perfect software. I use it with my Reeder App on my iPhone and cannot express how much this affects my personal and professional life. It is so useful to keep me informed i really love it.
Thank you so much to everybody who even participates a little bit in this project. This is a fantastic project!

**Comment by birgerj** [comment]
Just switched to FreshRSS from [paid rss service with nice app]. Really happy with it!

**Comment by jphastings** [comment]
I started using FreshRSS a year or so ago, after getting into Yunohost and looking for a self-hosted replacement for Feedly — _oh my days_ I wish I'd looked sooner! I don't use FreshRSS' UI directly often (I tend to use [Reeder](https://www.reederapp.com/)) but every time I do its simple, its easy and it makes sense.

Today was the first time I was a [little confused](https://github.com/FreshRSS/FreshRSS/issues/5727) by part of the interface, so I filed a bug — you got back to me _so quickly_, were kind and helpful and solved my issue right away. I'm so glad my regular donation is already set up!

Thank you for building — and crucially _maintaining_ — software so dependable and useful. A (very belated, possibly even 11th by now) happy birthday 🥳

**Comment by gullevek** [comment]
I used NewsBlur for 10 years, last year I had to setup my own server because the free riding ride on my friends home server came to an end because of the electricity prices. So I searched for RSS tools, found this one, installed it ... and it worked right out of the box. Imported my feeds, connect NetNewWire ... and last week canceled my NewsBlur subscription.

This software is amazing!

**Reply by Alkarex** [reply]
Welcome and thanks for the kind words 🙂

**Comment by lukasMega** [comment]
I just switched to **FreshRSS** ❤️ couple days ago. I'm very happy with it! 

I'm slowly stopping with 2 accounts on free **Inoreader** (because of limit max. 150 feeds).
What I like on Inoreader is feature Highlighters - for highlighting important words (see https://www.inoreader.com/blog/2015/03/new-feature-spot-most-important-terms.html)

With help of plugin CustomJS (+ CustomCSS) I was able to successfully add such highlighting feature into FreshRSS (using mark.js library https://github.com/julkue/mark.js/).

<details><summary>:arrow_lower_right: Click to show:</summary>

- I defined a bunch of strings to highlight, such as `Raspberry Pi`, `self-hosted`, `selfhosted`, ...
- and as you can see I'm using 2 different highlighting colors (stronger yellow and weaker yellow - with transparency)


dark:

<img width="1466" alt="Screen 2024-05-20 at 10 35 07" src="https://github.com/FreshRSS/FreshRSS/assets/2448004/b46ecd23-7e72-4341-8a2c-65cf7b063d48">


another theme:

<img width="1465" alt="Screen 2024-05-20 at 10 30 59" src="https://github.com/FreshRSS/FreshRSS/assets/2448004/82d5a6df-e81a-4228-9ac5-3c184fc1cc56">

</details>

- this is first working version, I need to polish it a bit, mainly css

---

**Now, this is definitely my most used self-hosted app!**
Thank you!

**Reply by Alkarex** [reply]
This could be a nice candidate for an extension :-)

**Reply by lukasMega** [reply]
Yeah, I was thinking about it, but i'm not sure when i will have a time for it, also i'm not very familiar with PHP (I have not used PHP since 2012).
But what I see as more realistic is to publish somewhere a short article how to add such word highlighting feature into FreshRSS :)

**Reply by Alkarex** [reply]
You could put a little description in https://github.com/FreshRSS/FreshRSS/discussions/categories/show-and-tell
